files/launch/WebAppMgr.service

```diff
@@ -24,4 +24,4 @@ Restart=on-failure
 RestartSec=50
```

Comment:
@jdapena is this still needed with this changes?

Reply:
Maybe not. Or at least a smaller value for regular restarts after crash could be ok.